regression/pubsub.js

```diff
@@ -198,9 +205,40 @@ describe('pubsub', function() {
         assert.equal(msg.data, 'hello');
         subscription.ack(msg.id, done);
       });
-      topic.publishRaw({
-        data: new Buffer('hello').toString('base64')
-      }, assert.ifError);
+      topic.publish([
```

Comment:
publishBatch returns messageIds, which can be used for tracking individual messages by subscribers. Is it possible to test whether the messageIds returned by publishBatch are the same as messageIds in pulled messages?